README.md

```diff
@@ -99,6 +99,7 @@ jobs:
           mode: qserver
           project: project-name
           viewerUrl: https://domain.com/tiobeweb/TICS/api/cfg?name=config
+          branchdir: ${{ GITHUB_WORKSPACE }}

```

Comment:
Is it recommended to set a path here, or is it better to use `.`?
As this option is also used by TICSQServer in later runs where this option has not been set.

Reply:
Does not really matter. To me  ${{ GITHUB_WORKSPACE }} is a bit more expressive/clear than just a dot. Also the dot can represent other locations than  ${{ GITHUB_WORKSPACE }} in case you CD to somewhere else. In the end it's up to the user to configure the correct location. I think  ${{ GITHUB_WORKSPACE }} is the most common way.